tests/test_browser.py

```diff
@@ -3542,13 +3542,15 @@ def test_separate_asm(self):
     for opts in [['-O0'], ['-O1'], ['-O2'], ['-O2', '--closure', '1']]:
       print(opts)
       open('src.cpp', 'w').write(self.with_report_result(open(path_from_root('tests', 'browser_test_hello_world.c')).read()))
-      Popen([PYTHON, EMCC, 'src.cpp', '-o', 'test.html', '-s', 'WASM=0'] + opts).communicate()
+      run_process([PYTHON, EMCC, 'src.cpp', '-o', 'test.html', '-s', 'WASM=0'] + opts)
       self.run_browser('test.html', None, '/report_result?0')
 
+      print('run one');
```

Comment:
were these debug printfs?

Reply:
Yeah, but it's nice to keep them, I think - they show progress through the test, if you're watching it run.

Reply:
Having debugged several tests that do this in test_core, they also help pinpoint which iteration of the test failed.